ansible/roles/omeka/tasks/main.yml

```diff
@@ -57,6 +53,10 @@
      - Search-By-Metadata-1.2.zip
    when: omeka_exists.stat.exists == false
 
+ - name: Set the permissions on the Omeka directory
+   file: path=/var/www/omeka owner=www-data group=www-data recurse=yes
+   when: omeka_exists.stat.exists == false
```

Comment:
Please don't make the entire folder owned by `www-data`. Only make necessary folders owned by www-data. This is potentially dangerous, especially for php apps.